src/uniformtypeidentifiers.cs

```diff
@@ -542,6 +542,10 @@ interface UTTypes {
 
 		[Field ("UTTypeLog")]
 		UTType Log { get; }
+
+		[Watch(10, 0), TV(17, 0), MacCatalyst(17, 0), Mac(14, 0), iOS(17, 0)]
+		[Field("UTTypeAHAP")]
+		UTType UTTypeAHAP { get; }
```

Comment:
I wasn't able to figure out what "AHAP" stands for, but I'm guessing it's an acronym, in which case only the first letter is capitalized:

```suggestion
		UTType UTTypeAhap { get; }
```

Reply:
now looking at it, could also modify the exposed name to match the others too